See: https://trac.macports.org/ticket/74442
See: https://chromium-review.googlesource.com/c/chromium/src/+/8025516

From 6c0a651f9cf91d07c87be8feba854a38a311aba6 Mon Sep 17 00:00:00 2001
From: Victor Hugo Vianna Silva <victorvianna@google.com>
Date: Wed, 01 Jul 2026 08:36:34 -0700
Subject: [PATCH] Fix macOS 27 build of `chrome` target

sandbox::Seatbelt::kProfilePureComputation is unused, so can be
removed.

Fixed: 530141405
Change-Id: Idcca8b7863c2b10821d8eae5bca782c80be6fe70
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/8025516
Auto-Submit: Victor Vianna <victorvianna@google.com>
Reviewed-by: Mark Mentovai <mark@chromium.org>
Commit-Queue: Mark Mentovai <mark@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1655528}
---

diff --git a/sandbox/mac/seatbelt.cc b/sandbox/mac/seatbelt.cc
index e9321ba..0ecb609 100644
--- src/3rdparty/chromium/sandbox/mac/seatbelt.cc.orig
+++ src/3rdparty/chromium/sandbox/mac/seatbelt.cc
@@ -139,12 +139,6 @@
   }
 }

-// Initialize the static member variables.
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-const char* Seatbelt::kProfilePureComputation = kSBXProfilePureComputation;
-#pragma clang diagnostic pop
-
 // static
 bool Seatbelt::Init(const char* profile, uint64_t flags, std::string* error) {
 // OS X deprecated these functions, but did not provide a suitable replacement,
diff --git a/sandbox/mac/seatbelt.h b/sandbox/mac/seatbelt.h
index 75dc798..76ab5cd 100644
--- src/3rdparty/chromium/sandbox/mac/seatbelt.h.orig
+++ src/3rdparty/chromium/sandbox/mac/seatbelt.h
@@ -84,8 +84,6 @@
   // Returns whether or not the process is currently sandboxed.
   static bool IsSandboxed();

-  static const char* kProfilePureComputation;
-
   Seatbelt(const Seatbelt& other) = delete;
   Seatbelt& operator=(const Seatbelt& other) = delete;

